src/test/java/javax/jmdns/impl/ServiceTypeDecoderTest.java

```diff
@@ -217,7 +218,7 @@ public void testReverseDNSQuery() {
     public void testAddress() {
         String type = "panoramix.local.";
 
-        Map<ServiceInfo.Fields, String> map = ServiceTypeDecoder.decodeQualifiedNameMapForType(type);
```

Comment:
Since you made the code use the OR operator to allow also the new recordType, this test became valid in more than 1 way. So please create a  new test cases that tests the TYPE_A and Type_AAA methods, but leave this case intact with the original method call.

Reply:
change removed